src/GitHubCreateMergePRs/Program.cs

```diff
@@ -13,6 +13,7 @@ public static async Task Main(string[] args)
         // Default when no args passed in.
         var isDryRun = true;
         var isAutomated = false;
+        var updateExistingPr = true;
```

Comment:
Should this default to 'false'?

Reply:
Actually, for simplicity it feels like this shouldn't be a command line arg, it should only be specified in config.xml per-repo. Do you agree?

Reply:
Sure, sounds reasonable.